integration_test/third_party_apps_data/applications/hbase/centos_rhel/install

```diff
@@ -0,0 +1,36 @@
+set -e
+
+sudo yum -y install \
+    java-1.8.0-openjdk java-1.8.0-openjdk-devel wget
+
+sudo mkdir /opt/hbase
+wget http://apache.mirror.gtcomm.net/hbase/stable/hbase-2.4.9-bin.tar.gz
+sudo tar -xvf hbase-2.4.9-bin.tar.gz -C /opt/hbase --strip 1
+
+JAVA_HOME="$(dirname "$(dirname "$(readlink "$(readlink "$(which javac)")")")")"
+
+cat <<EOF | sudo tee /etc/systemd/system/hbase.service
+[Unit]
+Description=HBase Hadoop database
+After=network.target
+
+[Service]
+Type=forking
```

Comment:
It's probably not important if this is reliable enough for a test, but AFAICT `start-hbase.sh` will start multiple daemons, but `Type=forking` expects only a single master process, so it's possible that systemd will be confused and `stop`/`restart` will misbehave.

Reply:
removed the service file and just using the env file as we only need to start it and stop it.

Reply:
Thanks!